drivers/gpu/drm/i915/i915_getparam.c

```diff
@@ -160,6 +160,10 @@ int i915_getparam_ioctl(struct drm_device *dev, void *data,
 	case I915_PARAM_PERF_REVISION:
 		value = i915_perf_ioctl_version();
 		break;
+        case I915_PARAM_IS_GVT:
+                value = intel_vgpu_active(i915);
+                DRM_INFO("=============Invoking GetParam GVT active, return: %d", intel_vgpu_active(i915));
```

Comment:
Is this log really needed or just for debug purpose?

Reply:
Just for debugging, I will update it soon

Reply:
updated